public/js/api.js

```diff
@@ -1,6 +1,91 @@
-var  clipboard = new Clipboard(".copy");
 
-clipboard.on("success", function() {
-    swal("Token copied successfully");
-});
+	var  clipboard = new Clipboard(".copy");
 
+	clipboard.on("success", function() {
+    	swal("Token copied successfully");
+	});
+
+	$("#delete-api").click(function() {
+		var id   = $(this).data("id");
```

Comment:
**Code Quality-Nitpick**
@andela-araimi The indentation here is looking unprofessional, please fix it.